submissions/unabyband/popup-task/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <link rel="stylesheet" href="css/main.css">
+        <title>unabyband pop-up</title>
+    </head>
+    <body>
+        <div class="header">
+            <div class="top_menu">
+                <div class="top_menu__item-container">
```

Comment:
list of items starts here, not only in popup

Reply:
Fixed